src/common/constants.js

```diff
@@ -1,22 +1,34 @@
-export const affiliate_signin_url = 'https://login.deriv.com/signin.php'
-export const affiliate_signup_url = 'https://login.deriv.com/signup.php'
+const isBrowser = () => typeof window !== 'undefined'
+
+const supported_domains = ['deriv.com', 'deriv.me']
```

Comment:
Will this cause problems for `localhost` environments?

Reply:
For localhost, it will open ''deriv.com'' url